include/nds/rsa.h

```diff
@@ -3,6 +3,12 @@
 //
 // Copyright (C) 2017 Dave Murphy (WinterMute)
 
+// DSi RSA functions
+
+/// @file nds/rsa.h
```

Comment:
You will also need to go to `include/nds.h` and add a new entry for this file or it won't show up in the index.